staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -546,16 +682,13 @@ func watchHandler(start time.Time,
 	name string,
 	expectedTypeName string,
 	setLastSyncResourceVersion func(string),
+	exitOnInitEventBookmark *bool,
 	clock clock.Clock,
 	errc chan error,
 	stopCh <-chan struct{},
 ) error {
 	eventCount := 0
 
```

Comment:
Let's set exitOnInitEvents.. to false (if it's non-nil) as a defence in depth here...